src/azul/__init__.py

```diff
@@ -702,9 +704,27 @@ def subscribe_to_dss(self):
     def state_machine_name(self, lambda_name):
         return config.qualified_resource_name(lambda_name)
 
+    def _concurrency(self, value: str, retry: bool) -> int:
+        values = [int(s) for s in value.split('/')]
+        try:
+            return one(values)
+        except ValueError:
```

Comment:
This is conflating `1` and `1/2/3`. Invalid syntax should lead to failure. I wouldn't use split.